### PR TITLE
Add support for Rocky Linux

### PR DESCRIPTION
**Pull request by NeilHanlon**
*Issue #, if available:* N/A

*Description of changes:* 

Add Rocky Linux to `SKIP_NO_LIBWRAP_RELEASES` so instances running it can mount EFS.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by lisenet** [comment]
Any ETA on when this can be merged?

**Comment by lukosbo** [comment]
Thanks for the PR. We’ll have someone take a look.

**Comment by Cappuccinuo** [comment]
Hey @NeilHanlon , thanks for the PR! Can you resolve the conflict based on current code base + combine the two commits into one commit? Thanks.

**Comment by NeilHanlon** [comment]
> Hey @NeilHanlon , thanks for the PR! Can you resolve the conflict based on current code base + combine the two commits into one commit? Thanks.

Totally - forgot to squash at first but should be good now.

Thank you!

**Review by Cappuccinuo — APPROVED** [review]

File: src/mount_efs/__init__.py [hunk 1]
```diff
@@ -265,6 +265,7 @@
 OPEN_SUSE_LEAP_RELEASE_NAME = "openSUSE Leap"
 SUSE_RELEASE_NAME = "SUSE Linux Enterprise Server"
 MACOS_BIG_SUR_RELEASE = "macOS-11"
+ROCKY8_RELEASE_NAME = 'Rocky Linux release 8'
```
Comment (Cappuccinuo) [inline src/mount_efs/__init__.py:268]: nit: Can you make this double quoted?

**Review by Cappuccinuo — CHANGES_REQUESTED** [review]
Minor format change. Single quote -> double quote

**Review by Cappuccinuo — APPROVED** [review]

**Comment by Cappuccinuo** [comment]
Merged, let us know if you encounter any issue while using this distribution.

**Pull request merged**